src/Aquifer.API/Endpoints/Feedback/BibleWell/Create/Endpoint.cs

```diff
@@ -14,13 +14,18 @@ public override void Configure()
 
     public override async Task HandleAsync(Request req, CancellationToken ct)
     {
+        var emailValue = (req.ContactType == FeedbackContactType.Email) ? req.ContactValue : req.Email;
+        var phoneValue = (req.ContactType == FeedbackContactType.Phone) ? req.ContactValue : req.Phone;
```

Comment:
What about this?

Reply:
I think this makes sense, but I would check that Email and Phone are null also. We don't want to overwrite a value that was explicitly entered.